Runtime/Haptics/AudioClipHapticPulser.cs

```diff
@@ -2,35 +2,40 @@
 {
     using UnityEngine;
     using System.Collections;
+    using Malimbe.MemberClearanceMethod;
```

Comment:
Alphabetical order.

Reply:
I've been putting them in order of string length (so it looks neat and pleasing to the eye :) )

Reply:
Heh, sadly I can't tell my auto-formatting of the IDE **that**.

Reply:
The problem with alphabetical order is it puts out the order of importance.

Unity
System
Malimbe
Zinnia
VRTK

Reply:
I was only suggesting that order inside the same root namespace, so it's still whatever order we want between those. We're not consistent so far anyway with this - there's plenty of using blocks not adhering to it which may be a result of me using auto-formatting as well as us doing refactoring and whole repo splits but then not going through all of these.

To me using statements are to make the compiler happy, I never look at them myself so I prefer automatic management of them.

Reply:
i'd say we dont bother updating stuff like namespaces in these PRs (as the other tracking PR which is huge will have loads of these cases). I suggest we do a code format clean up at the end